Formula/bleachbit.rb

```diff
@@ -0,0 +1,23 @@
+class Bleachbit < Formula
+  include Language::Python::Virtualenv
+
+  desc "Antiforensics and privacy system disk cleaner"
+  homepage "https://www.bleachbit.org"
+  url "https://github.com/bleachbit/bleachbit/archive/v4.0.0.tar.gz"
+  sha256 "5f7813b1ecda32647b1a31564b66f369fbd0d50d67924f256a0d414abc07501b"
+
+  depends_on "pygobject3"
+  depends_on "python@3.8"
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+    system "python3", "setup.py", "install", "--prefix=#{prefix}",
+                      "--single-version-externally-managed",
+                      "--record=installed.txt"
+    venv.pip_install_and_link buildpath
+  end
+
+  test do
+    assert_match "BleachBit version ", shell_output("#{bin}/bleachbit --sysinfo")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks @SMillerDev for the detailed review! I'll be happy to add more thorough and complete tests once I get BleachBit registered as a proper `bleachbit` executable into the system. There's lot of opportunity as we can run a couple of cleanup jobs directly by the way of the CLI.